test/OpenTelemetry.Exporter.Geneva.Tests/GenevaMetricExporterOptionsTests.cs

```diff
@@ -69,5 +69,24 @@ public void InvalidPrepopulatedDimensions()
             expectedErrorMessage = $"Value provided for the dimension: DimensionKey exceeds the maximum allowed limit of {GenevaMetricExporter.MaxDimensionValueSize} characters for dimension value.";
             Assert.Equal(expectedErrorMessage, invalidDimensionValueException.Message);
         }
+
+        [Fact]
+        public void InvalidMetricExportInterval()
+        {
+            var negativeInputMetricExportIntervalMillisecondsException = Assert.Throws<ArgumentOutOfRangeException>(() =>
+            {
+                var exporterOptions = new GenevaMetricExporterOptions
+                {
+                    MetricExportIntervalMilliseconds = 500,
+                };
+            });
+
+#if NET6_0 || NETCOREAPP3_1_OR_GREATER
+            var negativeInputExpectedErrorMessage = "Must be in the range: [1000, 2147483647] (Parameter 'value')\r\nActual value was 500.";
+#else
+            var negativeInputExpectedErrorMessage = "Must be in the range: [1000, 2147483647]\r\nParameter name: value\r\nActual value was 500.";
+#endif
+            Assert.Equal(negativeInputExpectedErrorMessage, negativeInputMetricExportIntervalMillisecondsException.Message);
```

Comment:
Do we really want this? My gut feeling is that the actual error message is owned by `Guard`, and if we really want to validate it, it should be in `GuardTests.cs`.

BTW, the use of EOL (CRLF vs. LF) might break depending on the operating system.

Reply:
So maybe only validating whether it's an ArgumentOutofRange sufficient here?
I am not sure whether we really need this test.

Reply:
I think we should definitely test that we throw an `ArgumentOutOfRangeException` when we provide a value less than 1000 ms.

@reyang How do we test that we have called the right `Guard` method with the right set of parameters? I think we could assert the message here as well to ensure that.

Reply:
I think the following tests should be good enough:
1. use 999ms, the ctor should complain
2. use 1000ms, the ctor should succeed